### PR TITLE
Create Dynamic Array

### PR DESCRIPTION
**Pull request by PaulAgeyev**
In this PR: 

Created a project with a root component which has an input ("Name") and a button ("Add"). When user clicks the button a new line of text appears ("Hello [[name]]") where name is a value of input.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dkalinin) [inline .idea/vcs.xml]: .idea should be added to .gitignore.
bower_components too.

**Comment by dkalinin** [comment]
@PaulAgeyev Also, please use more descriptive name for pull requests.

**Comment by PaulAgeyev** [comment]
@dkalinin, PTAL

File: .idea/vcs.xml [hunk 2]
```diff
@@ -3,4 +3,4 @@
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
-</project>
\ No newline at end of file
+</project>
```
Comment (dkalinin) [inline .idea/vcs.xml]: This file should be also removed.

**Pull request merged**